client/src/reducers/layoutChoice.ts

```diff
@@ -29,16 +28,6 @@ const LayoutChoice = (
   nextSharedState: RootState
 ): LayoutChoiceState => {
   switch (action.type) {
-    case "initial data load complete": {
-      const { layoutChoice } = action;
-
-      return {
-        ...state,
-        available: selectAvailableLayouts(nextSharedState),
```

Comment:
I think we might still need to update `available` in the state, since nowhere else is adding it 😄 ?

Reply:
right! fixing that now